web/modules/custom/rdf_entity/rdf_entity.module

```diff
@@ -398,3 +400,86 @@ function rdf_entity_theme_suggestions_rdf_entity(array $variables) {
 
   return $suggestions;
 }
+
+/**
+ * Returns the requirements related to virtuoso version.
+ *
+ * @return array
+ *   The virtuoso version requirements.
+ */
+function rdf_entity_virtuoso_version_requirements() {
+  $minimum_version = '07.00.0000';
+  /** @var \Drupal\rdf_entity\Database\Driver\sparql\Connection $connection */
+  $connection = Database::getConnection('default', 'sparql_default');
+  $client = Http::getDefaultHttpClient();
+  $client->resetParameters(TRUE);
+  $client->setUri($connection->getQueryUri());
+  $client->setMethod('GET');
+  $response = $client->request();
+  $server_header = $response->getHeader('Server');
+  preg_match('/Virtuoso\/(.*?)\s/', $server_header, $matches);
+  $version = (is_array($matches) && count($matches) === 2) ? $matches[1] : [];
+
+  $requirements = [
+    'rdf_entity_virtuoso_version' => [
+      'title' => t('Virtuoso version'),
+      'description' => t('Virtuoso version meets minimum requirements.'),
```

Comment:
The operator `+` on arrays appends if the keys are not yet existing, so nothing is overridden.
```
Virtuoso version meets minimum requirements.              [error]
```